aiven/client/speller.py

```diff
@@ -4,21 +4,24 @@
 # MIT license: www.opensource.org/licenses/mit-license.php
 
 
-def suggest(word_to_check, known_words):
-    def get_correction(word):
+from typing import Iterable, Optional
+
+
+def suggest(word_to_check: str, known_words: Iterable[str]) -> Optional[str]:
+    def get_correction(word: str) -> Optional[str] :
         """Most probable spelling correction for word."""
         candidates = get_candidates(word)
         return next(iter(candidates)) if candidates else None
 
-    def get_candidates(word):
+    def get_candidates(word: str) -> Iterable[str]:
         """Generate possible spelling corrections for word."""
         return get_known([word]) or get_known(get_edits1(word)) or get_known(get_edits2(word)) or []
 
-    def get_known(words):
+    def get_known(words: Iterable[str]) -> Iterable[str]:
         """The subset of `words` that appear in the dictionary of WORDS."""
         return set(w for w in words if w in known_words)
 
-    def get_edits1(word):
+    def get_edits1(word: str) -> Iterable[str]:
```

Comment:
Consider `Iterator[str]`

Reply:
Also here considering the return value I will keep it as `Iterable`.